dev/Common/TerminalVelocityFeatures-BadgeNotifications.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<features xmlns="http://microsoft.com/windowsappsdk/TerminalVelocity/20210729/TerminalVelocityFeatures.xsd">
+
+  <!-- Copyright (c) Microsoft Corporation. All rights reserved.
+
+     Licensed under the MIT License. See LICENSE in the project root for license information.
```

Comment:
move the copy right info to the top of the document

Reply:
This is the convention used for all the terminal velocity files in WindowsAppSdk